From: Your Name <you@example.com>
Date: Fri, 3 Feb 2023 15:31:39 +0000
Subject: 109 Fix Adblock

---
 chrome/browser/ui/tab_helpers.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -328,6 +328,8 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents) {
   }
 #endif
 
+  adblock::AdblockControllerFactory::GetForBrowserContext(
+      web_contents->GetBrowserContext());
   AdblockWebContentObserver::CreateForWebContents(
       web_contents,
       adblock::SubscriptionServiceFactory::GetForBrowserContext(
--
2.25.1
