From: Your Name <you@example.com>
Date: Mon, 6 Mar 2023 10:13:05 +0000
Subject: 111 temp fix build

---
 build/config/android/rules.gni | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -5083,8 +5083,8 @@ if (enable_java_templates && is_android) {
 
       if (!_proguard_enabled) {
         _dex_target = "${_module_target_name}__final_dex"
-        _rebased_module_build_config =
-            rebase_path(_module_build_config, root_build_dir)
+        # _rebased_module_build_config =
+        #     rebase_path(_module_build_config, root_build_dir)
         _dex_path = "$target_out_dir/$_module_target_name/$_module_target_name.mergeddex.jar"
         dex(_dex_target) {
           forward_variables_from(invoker,
--
2.25.1
