From: Your Name <you@example.com>
Date: Mon, 6 Mar 2023 10:13:05 +0000
Subject: 111 temp fix build

---
 build/config/android/rules.gni | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -5077,14 +5077,14 @@ if (enable_java_templates && is_android) {
     _all_module_unused_resources_deps = []
     foreach(_module, _modules) {
       _module_target = _module.module_target
-      _module_target_name = get_label_info(_module_target, "name")
       _module_build_config = _module.build_config
       _module_build_config_target = _module.build_config_target
 
       if (!_proguard_enabled) {
+        _module_target_name = get_label_info(_module_target, "name")
         _dex_target = "${_module_target_name}__final_dex"
-        _rebased_module_build_config =
-            rebase_path(_module_build_config, root_build_dir)
+        # _rebased_module_build_config =
+        #     rebase_path(_module_build_config, root_build_dir)
         _dex_path = "$target_out_dir/$_module_target_name/$_module_target_name.mergeddex.jar"
         dex(_dex_target) {
           forward_variables_from(invoker,
--
2.25.1
