From: Your Name <you@example.com>
Date: Wed, 12 Oct 2022 12:00:06 +0000
Subject: Add flag to configure maximum connecti (win fixup)

---
 chrome/browser/about_flags.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -1040,10 +1040,12 @@ const FeatureEntry::Choice kForceColorProfileChoices[] = {
      switches::kForceDisplayColorProfile, "hdr10"},
 };
 
+#if BUILDFLAG(IS_ANDROID)
 const FeatureEntry::Choice kMaxConnectionsPerHostChoices[] = {
     {features::kMaxConnectionsPerHostChoiceDefault, "", ""},
     {features::kMaxConnectionsPerHostChoice15, switches::kMaxConnectionsPerHost, "15"},
 };
+#endif
 
 const FeatureEntry::Choice kMemlogModeChoices[] = {
     {flags_ui::kGenericExperimentChoiceDisabled, "", ""},
--
2.25.1
