From: Your Name <you@example.com>
Date: Mon, 7 Nov 2022 16:10:21 +0000
Subject: Add menu item to view source (add fixup)

---
 .../app/appmenu/AppMenuPropertiesDelegateImpl.java       | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
@@ -1178,8 +1178,13 @@ public class AppMenuPropertiesDelegateImpl implements AppMenuPropertiesDelegate
     protected void updateViewSourceMenuItem(
             Menu menu, Tab currentTab) {
         MenuItem viewSourceMenuItem = menu.findItem(R.id.view_source_id);
-        String url = currentTab.getUrl().getSpec();
-        viewSourceMenuItem.setVisible(!url.isEmpty() && !url.startsWith("view-source:"));
+        if (currentTab != null) {
+            String url = currentTab.getUrl().getSpec();
+            viewSourceMenuItem.setVisible(!url.isEmpty() && !url.startsWith("view-source:"));
+        } else {
+            // currentTab maybe null with start surface
+            viewSourceMenuItem.setVisible(false);
+        }
     }
 
     /**
--
2.25.1
