From: Your Name <you@example.com>
Date: Thu, 13 Oct 2022 10:33:34 +0000
Subject: Client hints overrides (win fixup)

---
 content/common/user_agent.cc | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -323,7 +323,11 @@ std::string BuildUserAgentFromProduct(const std::string& product) {
 }
 
 std::string BuildModelInfo() {
-  return "SAMSUNG SM-G960U";
+  std::string model;
+#if BUILDFLAG(IS_ANDROID)
+  model = "SAMSUNG SM-G960U";
+#endif
+  return model;
 }
 
 #if BUILDFLAG(IS_ANDROID)
--
2.25.1
