From: uazo <uazo@users.noreply.github.com>
Date: Tue, 18 Apr 2023 14:17:19 +0000
Subject: Disable SHA1 Server Signature

---
 net/base/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -79,8 +79,8 @@ BASE_FEATURE(kUseDnsHttpsSvcbAlpn,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSHA1ServerSignature,
-             "SHA1ServerSignature",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             "SHA1ServerSignature",                  // disabled
+             base::FEATURE_DISABLED_BY_DEFAULT);     // in bromite
 
 BASE_FEATURE(kEnableTLS13EarlyData,
              "EnableTLS13EarlyData",
--
2.25.1
