From: uazo <uazo@users.noreply.github.com>
Date: Wed, 2 Aug 2023 15:15:48 +0000
Subject: Do not block the build for incompatible profiles

---
 .gitignore  | 1 -
 v8/BUILD.gn | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/.gitignore b/.gitignore
--- a/.gitignore
+++ b/.gitignore
@@ -345,7 +345,6 @@ vs-chromium-project.txt
 /ui/surface/surface.xml
 /ui/surface/surface_gpu_tests.xml
 /ui/webui/internal
-/v8
 /webkit/data
 /webpagereplay_logs/
 /x86-generic_out/
diff --git a/v8/BUILD.gn b/v8/BUILD.gn
--- a/v8/BUILD.gn
+++ b/v8/BUILD.gn
@@ -2293,7 +2293,7 @@ template("run_mksnapshot") {
 
         # Replace this with --warn-about-builtin-profile-data to see the full
         # list of builtins with incompatible profiles.
-        "--abort-on-bad-builtin-profile-data",
+        "--warn-about-builtin-profile-data",
       ]
     }
 
--
2.25.1
