From: uazo <uazo@users.noreply.github.com>
Date: Wed, 2 Aug 2023 15:15:48 +0000
Subject: Do not block the build for incompatible profiles

---
 v8/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v8/BUILD.gn b/v8/BUILD.gn
--- a/v8/BUILD.gn
+++ b/v8/BUILD.gn
@@ -2271,7 +2271,7 @@ template("run_mksnapshot") {
 
         # Replace this with --warn-about-builtin-profile-data to see the full
         # list of builtins with incompatible profiles.
-        "--abort-on-bad-builtin-profile-data",
+        "--warn-about-builtin-profile-data",
       ]
     }
 
--
2.25.1
