From: Your Name <you@example.com>
Date: Fri, 23 Sep 2022 11:26:07 +0000
Subject: Enable CrossSiteFlagNetworkAnonymizationKey flag

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -164,7 +164,7 @@ const base::Feature kEnableDoubleKeyNetworkAnonymizationKey{
 
 const base::Feature kEnableCrossSiteFlagNetworkAnonymizationKey{
     "EnableCrossSiteFlagNetworkAnonymizationKey",
-    base::FEATURE_DISABLED_BY_DEFAULT};
+    base::FEATURE_ENABLED_BY_DEFAULT};
 
 const base::Feature kExpectCTPruning{"ExpectCTPruning",
                                      base::FEATURE_ENABLED_BY_DEFAULT};
--
2.25.1
