From: Your Name <you@example.com>
Date: Fri, 23 Sep 2022 11:24:45 +0000
Subject: Enable ThirdPartyStoragePartitioning flag

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -366,7 +366,7 @@ const base::FeatureParam<bool> kStorageAccessAPIAutoDenyOutsideFPS{
 // by the top level site to reduce fingerprinting.
 BASE_FEATURE(kThirdPartyStoragePartitioning,
              "ThirdPartyStoragePartitioning",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAlpsParsing, "AlpsParsing", base::FEATURE_ENABLED_BY_DEFAULT);
 
--
2.25.1
