From: Your Name <you@example.com>
Date: Fri, 23 Sep 2022 11:24:45 +0000
Subject: Enable ThirdPartyStoragePartitioning flag

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -358,7 +358,7 @@ const base::FeatureParam<bool> kStorageAccessAPIAutoDenyOutsideFPS{
 // Enables partitioning of third party storage (IndexedDB, CacheStorage, etc.)
 // by the top level site to reduce fingerprinting.
 const base::Feature kThirdPartyStoragePartitioning{
-    "ThirdPartyStoragePartitioning", base::FEATURE_DISABLED_BY_DEFAULT};
+    "ThirdPartyStoragePartitioning", base::FEATURE_ENABLED_BY_DEFAULT};
 
 const base::Feature kAlpsParsing{"AlpsParsing",
                                  base::FEATURE_ENABLED_BY_DEFAULT};
--
2.25.1
