From: Your Name <you@example.com>
Date: Thu, 13 Oct 2022 10:34:52 +0000
Subject: Fix windows build

---
 chrome/browser/devtools/devtools_window.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/devtools/devtools_window.cc b/chrome/browser/devtools/devtools_window.cc
--- a/chrome/browser/devtools/devtools_window.cc
+++ b/chrome/browser/devtools/devtools_window.cc
@@ -80,6 +80,7 @@
 #include "ui/events/keycodes/dom/keycode_converter.h"
 #include "ui/events/keycodes/keyboard_code_conversion.h"
 #include "ui/events/keycodes/keyboard_codes.h"
+#include "net/cert/x509_certificate.h"
 
 // This should be after all other #includes.
 #if defined(_WINDOWS_)  // Detect whether windows.h was included.
--
2.25.1
