From: Your Name <you@example.com>
Date: Mon, 30 Jan 2023 15:20:53 +0000
Subject: Remove experimental-relative-c++-abi-vtables

---
 build/config/android/BUILD.gn | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -45,13 +45,6 @@ config("compiler") {
     # by shrinking the alignment gap between segments. This also causes all
     # segments to be mapped adjacently, which breakpad relies on.
     ldflags += [ "-Wl,-z,max-page-size=4096" ]
-
-    # Use non-standard (non-Itanium) ABI for Android64. This shrinks vtables
-    # in half and places them to .rodata, which greatly improves memory
-    # footprint. It should be safe, assuming all of C++ in Chromium is
-    # compiled with this flag.
-    cflags_cc = [ "-fexperimental-relative-c++-abi-vtables" ]
-    ldflags += [ "-fexperimental-relative-c++-abi-vtables" ]
   }
 
   if (current_cpu == "arm64") {
--
2.25.1
