From: Your Name <you@example.com>
Date: Wed, 12 Oct 2022 12:01:37 +0000
Subject: Restore Search Ready Omnibox flag (win fixup)

---
 chrome/browser/about_flags.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -8982,9 +8982,11 @@ const FeatureEntry kFeatureEntries[] = {
      flag_descriptions::kPervasiveSystemAccentColorDescription, kOsWin,
      SINGLE_VALUE_TYPE(switches::kPervasiveSystemAccentColor)},
 #endif
+#if BUILDFLAG(IS_ANDROID)
     {"enable-search-ready-omnibox", flag_descriptions::kSearchReadyOmniboxName,
      flag_descriptions::kSearchReadyOmniboxDescription, kOsAndroid,
      FEATURE_VALUE_TYPE(chrome::android::kSearchReadyOmniboxFeature)},
+#endif
 
     {"site-engagement",
      flag_descriptions::kSiteEngagementName,
--
2.25.1
