From: uazo <uazo@users.noreply.github.com>
Date: Mon, 17 Jul 2023 15:26:11 +0000
Subject: Show warnings on downloads over HTTP

---
 chrome/common/chrome_features.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -1136,9 +1136,9 @@ BASE_FEATURE(kTreatUnsafeDownloadsAsActive,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Show warnings on downloads not delivered over HTTPS.
-BASE_FEATURE(kInsecureDownloadWarnings,
-             "InsecureDownloadWarnings",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kInsecureDownloadWarnings,         // enabled by
+             "InsecureDownloadWarnings",        // default
+             base::FEATURE_ENABLED_BY_DEFAULT); // in bromite
 
 // TrustSafetySentimentSurvey
 #if !BUILDFLAG(IS_ANDROID)
--
2.25.1
