From: uazo <uazo@users.noreply.github.com>
Date: Mon, 5 Jun 2023 17:26:38 +0000
Subject: TEMP Enable MojoDirectSharedMemoryAndroid

---
 mojo/core/embedder/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mojo/core/embedder/features.cc b/mojo/core/embedder/features.cc
--- a/mojo/core/embedder/features.cc
+++ b/mojo/core/embedder/features.cc
@@ -25,7 +25,7 @@ const base::FeatureParam<bool> kMojoLinuxChannelSharedMemEfdZeroOnWake{
 
 BASE_FEATURE(kMojoPosixUseWritev,
              "MojoPosixUseWritev",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif  // BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_NACL) &&
         // !BUILDFLAG(MOJO_USE_APPLE_CHANNEL)
 
--
2.25.1
