From: Your Name <you@example.com>
Date: Sat, 3 Dec 2022 08:59:11 +0000
Subject: temporary fix libaom build error

---
 third_party/libaom/BUILD.gn | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/third_party/libaom/BUILD.gn b/third_party/libaom/BUILD.gn
--- a/third_party/libaom/BUILD.gn
+++ b/third_party/libaom/BUILD.gn
@@ -258,11 +258,11 @@ static_library("libaom") {
       cpu_arch_full == "arm-neon-cpu-detect") {
     deps += [ ":libaom_intrinsics_neon" ]
   }
-  if (current_cpu == "arm64" || current_cpu == "arm") {
-    # This is needed by all arm boards due to aom_arm_cpu_caps()
-    sources +=
-        [ "//third_party/libaom/source/libaom/aom_ports/arm_cpudetect.c" ]
-  }
+  # if (current_cpu == "arm64" || current_cpu == "arm") {
+  #   # This is needed by all arm boards due to aom_arm_cpu_caps()
+  #   sources +=
+  #       [ "//third_party/libaom/source/libaom/aom_ports/arm_cpudetect.c" ]
+  # }
   if (is_android) {
     deps += [ "//third_party/android_ndk:cpu_features" ]
   }
--
2.25.1
