From: uazo <uazo@users.noreply.github.com>
Date: Mon, 5 Jun 2023 17:26:38 +0000
Subject: TEMP Enable MojoDirectSharedMemoryAndroid

---
 mojo/core/embedder/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mojo/core/embedder/features.cc b/mojo/core/embedder/features.cc
--- a/mojo/core/embedder/features.cc
+++ b/mojo/core/embedder/features.cc
@@ -42,7 +42,7 @@ BASE_FEATURE(kMojoIpcz, "MojoIpcz", base::FEATURE_ENABLED_BY_DEFAULT);
 #if BUILDFLAG(IS_ANDROID)
 BASE_FEATURE(kMojoDirectSharedMemoryAndroid,
              "MojoDirectSharedMemoryAndroid",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif
 
 }  // namespace core
--
2.25.1
