From: uazo <uazo@users.noreply.github.com>
Date: Tue, 14 Mar 2023 15:58:00 +0000
Subject: WIN Disable TabHoverCard images

---
 chrome/browser/ui/ui_features.cc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/browser/ui/ui_features.cc b/chrome/browser/ui/ui_features.cc
--- a/chrome/browser/ui/ui_features.cc
+++ b/chrome/browser/ui/ui_features.cc
@@ -212,11 +212,7 @@ BASE_FEATURE(kTabGroupsSaveSyncIntegration,
 // https://crbug.com/928954
 BASE_FEATURE(kTabHoverCardImages,
              "TabHoverCardImages",
-#if BUILDFLAG(IS_MAC)
              base::FEATURE_DISABLED_BY_DEFAULT
-#else
-             base::FEATURE_ENABLED_BY_DEFAULT
-#endif
 );
 
 const char kTabHoverCardImagesNotReadyDelayParameterName[] =
--
2.25.1
