From: Your Name <you@example.com>
Date: Wed, 12 Oct 2022 12:00:43 +0000
Subject: WIN ADDTO Add flag to disable external intent re

---
 chrome/browser/about_flags.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3848,9 +3848,11 @@ const FeatureEntry kFeatureEntries[] = {
     {"ui-disable-partial-swap", flag_descriptions::kUiPartialSwapName,
      flag_descriptions::kUiPartialSwapDescription, kOsAll,
      SINGLE_DISABLE_VALUE_TYPE(switches::kUIDisablePartialSwap)},
+#if BUILDFLAG(IS_ANDROID)
     {"disable-external-intent-requests", flag_descriptions::kDisableExternalIntentRequestsName,
      flag_descriptions::kDisableExternalIntentRequestsDescription, kOsAll,
      SINGLE_DISABLE_VALUE_TYPE("disable-external-intent-requests")},
+#endif
     {"disable-webrtc-hw-decoding", flag_descriptions::kWebrtcHwDecodingName,
      flag_descriptions::kWebrtcHwDecodingDescription, kOsAndroid | kOsCrOS,
      SINGLE_DISABLE_VALUE_TYPE(switches::kDisableWebRtcHWDecoding)},
--
2.25.1
