From: Your Name <you@example.com>
Date: Mon, 19 Dec 2022 11:23:10 +0000
Subject: WIN ADDTO Add AllowUserCertificates flag

---
 chrome/browser/about_flags.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -9226,10 +9226,12 @@ const FeatureEntry kFeatureEntries[] = {
      flag_descriptions::kHttpsUpgradesDescription, kOsDesktop | kOsAndroid,
      FEATURE_VALUE_TYPE(features::kHttpsUpgrades)},
 
+#if BUILDFLAG(IS_ANDROID)
     {"allow-user-certificates",
      flag_descriptions::kAllowUserCertificatesName,
      flag_descriptions::kAllowUserCertificatesDescription, kOsAndroid,
      FEATURE_VALUE_TYPE(chrome::android::kAllowUserCertificates)},
+#endif
     {"omnibox-updated-connection-security-indicators",
      flag_descriptions::kOmniboxUpdatedConnectionSecurityIndicatorsName,
      flag_descriptions::kOmniboxUpdatedConnectionSecurityIndicatorsDescription,
--
2.25.1
