From: Your Name <you@example.com>
Date: Tue, 14 Feb 2023 16:41:42 +0000
Subject: WIP disable visited pseudo class

---
 components/visitedlink/common/visitedlink_common.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/visitedlink/common/visitedlink_common.cc b/components/visitedlink/common/visitedlink_common.cc
--- a/components/visitedlink/common/visitedlink_common.cc
+++ b/components/visitedlink/common/visitedlink_common.cc
@@ -43,6 +43,7 @@ bool VisitedLinkCommon::IsVisited(const GURL& url) const {
 }
 
 bool VisitedLinkCommon::IsVisited(Fingerprint fingerprint) const {
+  if ((true)) return false;
   // Go through the table until we find the item or an empty spot (meaning it
   // wasn't found). This loop will terminate as long as the table isn't full,
   // which should be enforced by AddFingerprint.
--
2.25.1
