From: Your Name <you@example.com>
Date: Wed, 12 Oct 2022 11:59:30 +0000
Subject: WIN ADDTO Revert flags: remove num-raster-thre

---
 chrome/browser/about_flags.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -383,6 +383,7 @@ const FeatureEntry::Choice kOverlayStrategiesChoices[] = {
      "single-fullscreen,single-on-top,underlay"},
 };
 
+#if BUILDFLAG(IS_ANDROID)
 const FeatureEntry::Choice kNumRasterThreadsChoices[] = {
     {flags_ui::kGenericExperimentChoiceDefault, "", ""},
     {flag_descriptions::kNumRasterThreadsOne, cc::switches::kNumRasterThreads, "1"},
@@ -391,6 +392,7 @@ const FeatureEntry::Choice kNumRasterThreadsChoices[] = {
      "3"},
     {flag_descriptions::kNumRasterThreadsFour, cc::switches::kNumRasterThreads,
      "4"}};
+#endif
 
 const FeatureEntry::Choice kTouchTextSelectionStrategyChoices[] = {
     {flags_ui::kGenericExperimentChoiceDefault, "", ""},
--
2.25.1
